### ISSUE TITLE
Inconsistent Uppercase Font in Terminal Mode

### ISSUE THREAD

**Issue by Tenebryo** [jamwaffles/ssd1306#135]
- Version of `ssd1306` in use (if applicable): `0.4.0`
- MCU/other hardware in use: atmega328p
- Display resolution and interface: [I2C], [128x64]

## Description of the problem/feature request/other

Several uppercase characters in terminal mode are bold, inconsistent with the rest. The characters in question are `DLORW`, which use 2-3 pixel font instead of 1 pixel like the rest of the characters.

## Example code and result:

```rust
let interface = I2CDIBuilder::new().init(i2c);
let mut disp: TerminalMode<_> = Builder::new().connect(interface).into();

disp.init().unwrap();

disp.clear().unwrap();

for c in "!\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|".chars() {
    disp.print_char(c);
}
```
![oled_font](https://user-images.githubusercontent.com/6086428/93272468-63bac500-f76a-11ea-988c-db530749d262.jpg)

**Comment by jamwaffles** [comment]
In a sense this is by design; the font used renders those characters that way but I do agree it looks strange. We could certainly switch to a different font that's more consistent though. @therealprof wrote the initial terminalmode impl, so he might have a reason the current font was chosen. Font suggestions are welcome. I'd be happy to write a lil conversion script for one of the fonts in embedded-graphics if there's no other preference.

**Comment by therealprof** [comment]
No specific reason why I picked this font. Happy to see it replaced by a better one. Do we have a nice 7x7 (or smaller) font somewhere?

**Comment by jamwaffles** [comment]
I've switched to a new font which looks like this: 

![](https://user-images.githubusercontent.com/800791/100738146-1ba45980-33cd-11eb-9471-dbd1f59e33a7.jpeg)

This will go out in the next release.

**Comment by jamwaffles** [comment]
Released in [0.4.1](https://crates.io/crates/ssd1306/0.4.1).
